### PR TITLE
[zh] Resync page distributed-tracing/lightstep

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
Resync page distributed-tracing/lightstep.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 1]
```diff
@@ -1,58 +1,69 @@
 ---
-title: LightStep
-description: 怎样配置代理才能把追踪请求发送到 LightStep。
+title: Lightstep
+description: 怎样配置代理才能把追踪请求发送到 Lightstep
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md:3]:
```suggestion
description: 怎样配置代理才能把追踪请求发送到 Lightstep。
```

**Review by windsonsea — COMMENTED** [review]
Check [lint errors](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/12635/lint_istio.io/1624564535383822336) please.

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 2]
```diff
@@ -1,58 +1,69 @@
...
 
 此任务使用 [Bookinfo](/zh/docs/examples/bookinfo/) 的样例代码作为示例。
 
 ## 开始之前{#before-you-begin}
 
-1. 确保你有一个 LightStep 账户。这里可以免费[注册](https://lightstep.com/products/tracing/)试用 LightStep Tracing，或者[联系 LightStep](https://lightstep.com/contact/) 创建企业级的 LightStep [𝑥]PM 账户。
+1. 确保你有一个 Lightstep 账户。这里可以免费[注册](https://lightstep.com/products/tracing/)试用 Lightstep。
+
+1. 如果您使用的是[本地 Satellite](https://docs.lightstep.com/docs/learn-about-satellites#on-premise-satellites), 请确保您有一个配置了 TLS 证书的 Satellite 池和一个公开的安全 GRPC 端口。
+   请参阅[安装和配置 Satellite](https://docs.lightstep.com/docs/install-and-configure-satellites) 获取更多有关设置 Satellite 的细节。
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md:27]:
```suggestion
    请参阅[安装和配置 Satellite](https://docs.lightstep.com/docs/install-and-configure-satellites) 获取更多有关设置 Satellite 的细节。
```

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 3]
```diff
@@ -1,58 +1,69 @@
...
+
+   对于 [Lightstep 公共 Satellite](https://docs.lightstep.com/docs/learn-about-satellites#public-satellites) 或 [开发者模式 Satellite](https://docs.lightstep.com/docs/learn-about -satellites#developer-satellites)，您的 Satellite 已经配置好了。 但是，您需要将[此证书](https://docs.lightstep.com/docs/instrument-with-istio-as-your-service-mesh#cacertpem-file)下载到本地目录。
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md:29]:
```suggestion
    对于 [Lightstep 公共 Satellite](https://docs.lightstep.com/docs/learn-about-satellites#public-satellites) 或 [开发者模式 Satellite](https://docs.lightstep.com/docs/learn-about -satellites#developer-satellites)，您的 Satellite 已经配置好了。但是，您需要将[此证书](https://docs.lightstep.com/docs/instrument-with-istio-as-your-service-mesh#cacertpem-file)下载到本地目录。
```

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 4]
```diff
@@ -88,9 +140,15 @@ LightStep 可以分析来自大规模生产级软件的 100% 未采样的事务
     $ curl http://$GATEWAY_URL/productpage
     {{< /text >}}
 
-1. 打开 LightStep [web UI](https://app.lightstep.com/)。
+## 可视化追踪数据{#visualize-trace-data}
 
-1. 导航到 Explorer 。
+1. 打开 Lightstep [web UI](https://app.lightstep.com/)。您会在服务目录中看到三个 Bookinfo 服务。
+
+    {{< image link="./istio-services.png" caption="Bookfinder services in the Service Directory" >}}
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md:147]:
```suggestion
    {{< image link="./istio-services.png" caption="Bookfinder services in the Service Directory" >}}
```
This image is missing

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 5]
```diff
@@ -1,58 +1,69 @@
...
-    - `global.tracer.lightstep.address="<satellite-address>"`
-    - `global.tracer.lightstep.accessToken="<access-token>"`
-    - `global.tracer.lightstep.secure=true`
-    - `global.tracer.lightstep.cacertPath="/etc/lightstep/cacert.pem"`
 
-    当执行安装命令时，可以使用 `--set key=value` 语法来配置这些参数，例如：
+   - `global.proxy.tracer="lightstep"`
+   - `meshConfig.defaultConfig.tracing.sampling=100`
+   - `meshConfig.defaultConfig.tracing.lightstep.address="<satellite-address>"`
+   - `meshConfig.defaultConfig.tracing.lightstep.accessToken="<access-token>"`
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md:48]:
```suggestion
    - `global.proxy.tracer="lightstep"`
    - `meshConfig.defaultConfig.tracing.sampling=100`
    - `meshConfig.defaultConfig.tracing.lightstep.address="<satellite-address>"`
    - `meshConfig.defaultConfig.tracing.lightstep.accessToken="<access-token>"`
```

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 6]
```diff
@@ -1,58 +1,69 @@
...
-        --set values.global.tracer.lightstep.secure=true \
-        --set values.global.tracer.lightstep.cacertPath="/etc/lightstep/cacert.pem"
+   当执行安装命令时，可以使用 `--set key=value` 语法来配置这些参数，例如：
+
+   {{< text bash >}}
+   $ istioctl install \
+       --set global.proxy.tracer="lightstep" \
+       --set meshConfig.defaultConfig.tracing.sampling=100 \
+       --set meshConfig.defaultConfig.tracing.lightstep.address="<satellite-address>" \
+       --set meshConfig.defaultConfig.tracing.lightstep.accessToken="<access-token>" \
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md:57]:
```suggestion
    当执行安装命令时，可以使用 `--set key=value` 语法来配置这些参数，例如：

    {{< text bash >}}
    $ istioctl install \
        --set global.proxy.tracer="lightstep" \
        --set meshConfig.defaultConfig.tracing.sampling=100 \
        --set meshConfig.defaultConfig.tracing.lightstep.address="<satellite-address>" \
        --set meshConfig.defaultConfig.tracing.lightstep.accessToken="<access-token>" \
    ```

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 7]
```diff
@@ -1,58 +1,69 @@
...
-1. 把 satellite 池证书颁发机构发的证书作为一个密钥存储在默认的命名空间下。
-    对于 LightStep Tracing 用户，要在这里下载并使用[这个证书](https://docs.lightstep.com/docs/instrument-with-istio-as-your-service-mesh)。
-    如果你把 Bookinfo 应用程序部署在了其它的命名空间下，就要在对的应命名空间下创建相应的密钥证书。
+### 使用公共或开发者模式 Satellite 部署 Istio{#deploy-istio-with-public-or-developer-mode-satellites}
+
+如果您使用的是公共或开发者模式 Satellite，或者如果您使用的是带有 TLS 证书的本地 Satellite，请按照这些步骤操作。
+
+1. 把 Satellite 池证书颁发机构发的证书作为一个密钥存储在默认的命名空间下。
+   对于 Lightstep Tracing 用户，要在这里下载并使用[这个证书](https://docs.lightstep.com/docs/instrument-with-istio-as-your-service-mesh)。
+   如果你把 Bookinfo 应用程序部署在了其它的命名空间下，就要在对的应命名空间下创建相应的密钥证书。
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md:66]:
```suggestion
    对于 Lightstep Tracing 用户，要在这里下载并使用[这个证书](https://docs.lightstep.com/docs/instrument-with-istio-as-your-service-mesh)。
    如果你把 Bookinfo 应用程序部署在了其它的命名空间下，就要在对的应命名空间下创建相应的密钥证书。
```

**Review by windsonsea — COMMENTED** [review]
These are lint errors:
- Indent 4 spaces
- Copy images while localize a file

File: content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md [hunk 8]
```diff
@@ -1,57 +1,67 @@
...
-1. 把 satellite 池证书颁发机构发的证书作为一个密钥存储在默认的命名空间下。
-    对于 LightStep Tracing 用户，要在这里下载并使用[这个证书](https://docs.lightstep.com/docs/instrument-with-istio-as-your-service-mesh)。
+	
+	    {{< text bash >}}
+	    $ istioctl install \
+	        --set global.proxy.tracer="lightstep" \
+	        --set meshConfig.defaultConfig.tracing.sampling=100 \
+	        --set meshConfig.defaultConfig.tracing.lightstep.address="<satellite-address>" \
+	        --set meshConfig.defaultConfig.tracing.lightstep.accessToken="<access-token>" \
+      {{< /text >}}
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/distributed-tracing/lightstep/index.md:58]:
```suggestion
    {{< text bash >}}
    $ istioctl install \
        --set global.proxy.tracer="lightstep" \
        --set meshConfig.defaultConfig.tracing.sampling=100 \
        --set meshConfig.defaultConfig.tracing.lightstep.address="<satellite-address>" \
        --set meshConfig.defaultConfig.tracing.lightstep.accessToken="<access-token>" \
    {{< /text >}}
```

**Review by windsonsea — COMMENTED** [review]
The indentation still has errors. You can direclty copy my suggested.

**Review by windsonsea — APPROVED** [review]
lgtm

**Pull request merged**